CMakeLists.txt

```diff
@@ -66,6 +66,17 @@ if(NOT CMAKE_BUILD_TYPE IN_LIST BUILD_TYPES)
     message(FATAL_ERROR "Unknown Build Type: ${CMAKE_BUILD_TYPE}")
 endif()
 
+# Flag to enforce colored output. When compiler recognize output to terminals, recent version automatically
+# set colored output. However, when using Ninja, the output is not directly the terminal. With this flag enabled,
+# also Ninja build will have colored outputs.
+if (${FORCE_COLORED_OUTPUT})
```

Comment:
Most (all?) other flags start with `ENABLE`. But `FORCE` makes still more sense here, right?

Reply:
With your idea of checking of the generator, we might get rid of the flag altogether.